docs/DeveloperGuide.md

```diff
@@ -665,107 +686,117 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+1. User requests to clear all patients.
+2. DocTrack clears all patients.
 
-    * 3b1. DocTrack shows an error message.
+    Use case ends.
 
-      Use case resumes at step 2.
+<br>
 
-**Use case: Duplicate an appointment**
+#### Use case: Add an appointment
 
 **MSS**
 
-1.  User requests to list appointments.
-2.  DocTrack shows a list of appointments.
-3.  User requests to duplicate a specific appointment in the list on a new date.
-4.  DocTrack duplicates appointment.
+1. User requests to add an appointment.
+2. DocTrack adds the appointment.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
-
-  Use case ends.
+* 1a. User enters invalid parameters.
 
-* 3a. The given index is invalid.
+    * 1a1. DocTrack shows an error message.
 
-    * 3a1. DocTrack shows an error message.
+      Use case resumes at step 1.
 
-      Use case resumes at step 2.
+* 1b. User enters an appointment that already exists.
 
-* 3b. The appointment details are invalid (i.e. wrongly formatted or overlap with existing appointment)
+    * 1b1. DocTrack shows an error message.
 
-    * 3b1. DocTrack shows an error message.
+      Use case resumes at step 1.
 
-      Use case resumes at step 2.
+    
+<br>
 
-**Use case: Update an appointment**
+#### Use case: Edit an appointment
 
 **MSS**
 
-1.  User requests to list appointments.
-2.  DocTrack shows a list of appointments.
-3.  User requests to update a specific appointment in the list with new details
-4.  DocTrack updates the appointment.
+1. User requests to edit a specific appointment in the list with new details.
+2. DocTrack updates the appointment with the new details.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The given index is invalid.
 
-  Use case ends.
+    * 1a1. DocTrack shows an error message.
+
+      Use case resumes at step 1.
+* 1b. The new appointment details are invalid.
+
+    * 1b1. DocTrack shows an error message.
+
+      Use case resumes at s
+
+<br>
+
+#### Use case: Delete an appointment
+
+**MSS**
+
+1. User requests to delete a specific appointment in the list.
+2. DocTrack deletes the appointment.
 
-* 3a. The given index is invalid.
+    Use case ends.
 
-    * 3a1. DocTrack shows an error message.
+**Extensions**
 
-      Use case resumes at step 2.
+* 1a. The given index is invalid.
 
-* 3b. The new details are invalid (i.e. wrongly formatted).
+    * 1a1. DocTrack shows an error message.
 
-    * 3b1. DocTrack shows an error message.
+      Use case resumes at step 1.
 
-      Use case resumes at step 2.
+<br>
 
-**Use case: Delete an appointment**
+#### Use case: View all appointments
 
 **MSS**
 
-1.  User requests to list appointments.
-2.  DocTrack shows a list of appointments.
-3.  User requests to delete a specific appointment in the list.
-4.  DocTrack deletes the appointment.
+1. User requests to list all appointments.
+2. DocTrack shows a list of appointments.
```

Comment:
Currently pending team discussion on whether or not to keep steps 1 and 2 about listing.

Reply:
Ok I've modified it to include the list use case.